Cargo.toml

```diff
@@ -6,16 +6,17 @@ authors = ["Jeremy Soller <jackpot51@gmail.com>"]
 build = "build.rs"
 
 [lib]
-name = "distinst"
-crate-type = ["lib", "dylib"]
+crate-type = ["dylib"]
```

Comment:
why should it be **only** dylib?

Reply:
Causes a build failure otherwise.

Reply:
Seems the fix for this particular issue is to ensure that the project is built by first specifying to only build the library, and then following up to build only the binary. Specifying to build both at the same time when changes are made will cause a build failure.